README.md

```diff
@@ -71,4 +72,22 @@ gzip -d GoogleNews-vectors-negative300.bin.gz
 python bin2txt.py GoogleNews-vectors-negative300.bin GoogleNews-vectors-negative300.txt 
 ```
 
+## References  
+
+Please cite the following works appropriately upon the usage of this repository:
+
+    @inproceedings{adhikari2019rethinking,
+    title={Rethinking Complex Neural Network Architectures for Document Classification},
+    author={Adhikari, Ashutosh and Ram, Achyudh and Tang, Raphael and Lin, Jimmy},
+    booktitle={NAACL},
+    year={2019}
+    }
+
+    @inproceedings{ram2019docbert,
```

Comment:
Should have arXiv identifier, e.g., arXiv:123123 (see Google Scholar examples).

Reply:
Will add this in a new PR.